.github/workflows/c8run-linux-x86-build.yaml

```diff
@@ -0,0 +1,73 @@
+name: "test c8run on linux"
+
+on:
+  push:
+    paths:
+      - "c8run/**"
+      - ".github/workflows/c8run-linux-x86-build.yaml"
+
+jobs:
+  test_c8run:
+    name: Test deploying camunda platform on c8run
+    runs-on: ubuntu-latest
```

Comment:
If the Windows job proves to be to dissimilar to Linux/Mac, then it can still be merged into the same workflow file as a job with a distinct different name (in addition to 3 matrix jobs).

Reply:
I think I've made the changes you've suggested here. Now I'll wait for the CI and fix any issues that come up.

Reply:
Thanks a lot, look good from my side